### PR TITLE
get_sales and get_earnings optimizations

### PR DESCRIPTION
**Pull request by chriscct7**
Noticed the double calls to get_post_meta. Can be easily optimized for the two functions which are called quite a bit.

### CODE REVIEW

**Comment by pippinsplugins** [comment]
Milestoned for 2.5.5

File: includes/class-edd-download.php [hunk 1]
```diff
@@ -575,12 +575,13 @@ public function get_button_behavior() {
 	public function get_sales() {
 
 		if( ! isset( $this->sales ) ) {
-
-			if ( '' == get_post_meta( $this->ID, '_edd_download_sales', true ) ) {
+			$sales = get_post_meta( $this->ID, '_edd_download_sales', true );
+			if ( '' ==  ) {
```
Comment (pippinsplugins) [inline includes/class-edd-download.php]: Missing `$sales` here.

**Comment by chriscct7** [comment]
Ready to go

**Comment by pippinsplugins** [comment]
@chriscct7 Can you please go ahead and resubmit to `release/2.6`?

**Pull request closed without merging**